boskos/boskos.go

```diff
@@ -0,0 +1,383 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"sync"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+)
+
+var configPath = flag.String("config", "resources.json", "Path to init resource file")
+var storage = flag.String("storage", "", "Path to presistent volume to load the state")
+
+func main() {
+	flag.Parse()
+	logrus.SetFormatter(&logrus.JSONFormatter{})
+
+	r, err := NewRanch(*configPath, *storage)
+	if err != nil {
+		logrus.WithError(err).Fatal("Fail to create my ranch!")
+	}
+
+	http.Handle("/", handleDefault())
+	http.Handle("/start", handleStart(r))
+	http.Handle("/done", handleDone(r))
+	http.Handle("/reset", handleReset(r))
+	http.Handle("/update", handleUpdate(r))
+	http.Handle("/metric", handleMetric(r))
+
+	go func() {
+		for range time.Tick(time.Minute) {
+			r.logStatus()
+			r.saveState()
+		}
+	}()
+
+	logrus.Info("Start Service")
+	logrus.WithError(http.ListenAndServe(":8080", nil)).Fatal("ListenAndServe returned.")
+}
+
+func handleDefault() http.HandlerFunc {
```

Comment:
Ok. Why do we need to verify boskos exists? Could this do something more useful... maybe return the current state of the ranch?